examples/with-sentry/components/withSentry.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Raven from 'raven-js'
+
+function withSentry (Child) {
```

Comment:
This HOC should also receive a component for the error case

Reply:
@sergiodxa I've tried adding a second component but it seems to not being re-rendering when the error is catched. Do you know what could be wrong here?

I've tried:
```jsx
if (this.state.error) {
    return <ErrorChild error={this.state.error} {...this.props} />
} else {
    return <Child {...this.props} />
}
```

Like if it only runs one time on loading.